js2.4.4.js

```diff
@@ -0,0 +1 @@
+const compose = (f, g) => (...args) => f(g(...args));
```

Comment:
add function invoke and console logs to explain about compose . follow naming conventions.. 
I want examples which uses this function